src/shared/components/annotation/OncoKB.spec.tsx

```diff
@@ -181,9 +183,40 @@ describe('OncoKB', () => {
         });
         array = [queryB, queryA];
         sortedArray = lazyMobXTableSort<IndicatorQueryResp>(array, OncoKB.sortValue, true);
-        assert.isAbove(
+        assert.isBelow(
             sortedArray.indexOf(queryA), sortedArray.indexOf(queryB),
             'variantExist test 2');
+
+        // Is Hotspot does not have any impact any more
+        queryA = initQueryIndicator({
+            oncogenic: 'Oncogenic',
+            hotspot: false
+        });
+        queryB = initQueryIndicator({
+            oncogenic: 'Oncogenic',
+            hotspot: true,
+            highestSensitiveLevel: 'LEVEL_2A'
+        });
+        array = [queryB, queryA];
+        sortedArray = lazyMobXTableSort<IndicatorQueryResp>(array, OncoKB.sortValue, true);
+        assert.isBelow(
+            sortedArray.indexOf(queryA), sortedArray.indexOf(queryB),
+            'isHotspot test 1');
```

Comment:
Having a descriptive assertion message might be better here: something like, `A(LEVEL_2A) should be higher than B(...)`